src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -34,34 +34,69 @@
 import org.takes.http.FtRemote;
 import org.takes.rq.RqFake;
 import org.takes.rq.RqHref;
+import org.takes.rq.RqMethod;
 import org.takes.rs.RsPrint;
 import org.takes.rs.RsText;
 
 /**
  * Test case for {@link TkProxy}.
  * @author Dragan Bozanovic (bozanovicdr@gmail.com)
+ * @author sebing
  * @version $Id$
  * @since 0.25
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
- * @todo #377:30min/DEV We need more tests for TkProxy.
- *  The tests should verify the different HTTP methods (GET, POST, etc),
- *  as well as the different combinations of request/response headers.
- */
+  */
 public final class TkProxyTest {
 
     /**
-     * TkProxy can work.
-     * @throws Exception If some problem inside
+     * TkProxy Testcase for testing the creation of the object with method POST.
```

Comment:
@sebing please update java doc according to this article www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names

same below